packages/cozy-app-publish/test/prepublish.spec.js

```diff
@@ -67,3 +70,13 @@ describe('Prepublish script', () => {
     expect(downcloudSpy).toHaveBeenCalled()
   })
 })
+
+if (process.env.TEST_INTEGRATION) {
```

Comment:
Behind a environment variable since it needs to download a file and is potentially long.

Reply:
👍